math-scala/src/main/scala/org/apache/mahout/math/drm/logical/CheckpointAction.scala

```diff
@@ -44,5 +44,6 @@ abstract class CheckpointAction[K: ClassTag] extends DrmLike[K] {
     case Some(cp) => cp
   }
 
+  val classTag = implicitly[ClassTag[K]]
```

Comment:
maybe we should expose that in DrmLike. 
I so far had a need for this in logical operators (AbstractBinaryOp$classTagK, AbstractUnaryOp$classTagK) for optimizer's needs. so maybe it needs to be promoted up if it turns out every implementation must keep implicit classtag inside.